api/clamav_scanner/clamav.py

```diff
@@ -0,0 +1,205 @@
+import datetime
+import hashlib
+import os
+import pwd
+import re
+import subprocess
+
+import botocore
+from pytz import utc
+
+from .common import AV_DEFINITION_S3_PREFIX
+from .common import AV_DEFINITION_PATH
+from .common import AV_DEFINITION_FILE_PREFIXES
+from .common import AV_DEFINITION_FILE_SUFFIXES
+from .common import AV_SIGNATURE_OK
+from .common import AV_SIGNATURE_UNKNOWN
+from .common import AV_STATUS_CLEAN
+from .common import AV_STATUS_INFECTED
+from .common import CLAMAVLIB_PATH
+from .common import CLAMSCAN_PATH
+from .common import FRESHCLAM_PATH
+from .common import create_dir
+
+from boto3wrapper.wrapper import get_session
+
+
+RE_SEARCH_DIR = r"SEARCH_DIR\(\"=([A-z0-9\/\-_]*)\"\)"
```

Comment:
Could this be compiled once here instead of each time it's used?
```python
RE_SEARCH_DIR = re.compile(r"SEARCH_DIR\(\"=([A-z0-9\/\-_]*)\"\)")
```

Reply:
that's a really good idea!